src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -55,12 +57,20 @@ public static boolean isValidFormat(String birthday) {
     public static boolean isValidDate(String birthday) {
         try {
             LocalDate.parse(birthday, BIRTHDATE_FORMATTER.withResolverStyle(ResolverStyle.STRICT));
-            return !isFutureDate(birthday);
+            return true;
```

Comment:
I dont think isFutureDate should be removed, this will allow future birthdays to be saved in the contacts

Reply:
If im not wrong this is already checked under `constructor`

Reply:
Oh I changed this because I realised that you had two seperate messages for future dates and valid date so I didn't want to check future dates here cos it wont return the future dates in that case.

Reply:
So j to clarify then, future dates are still illegal right, they wont be added to the contacts?

Reply:
Yeahhh this was your original implementation actually